air/benches/computing_op_flags.rs

```diff
@@ -0,0 +1,20 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use miden_air::stack::op_flags::{generate_evaluation_frame, OpFlags};
+use std::time::Duration;
+
+fn computing_op_flags(c: &mut Criterion) {
+    let mut group = c.benchmark_group("computing_op_flags");
```

Comment:
minor nit: let's rename the benchmark (including the name of the file) into `compute_op_flags`.